packages/SwingSet/src/kernel/vat-admin-hooks.js

```diff
@@ -112,5 +112,19 @@ export function makeVatAdminHooks(tools) {
       terminateVat(vatID, true, reasonCD);
       return harden({ body: stringify(undefined), slots: [] });
     },
+
+    changeOptions(argsCapData) {
+      // marshal([vatID, options]) -> null
+      const args = JSON.parse(argsCapData.body);
```

Comment:
probably appropriate to assert that `slots.length === 0`, if someone provides a vref as an option value then we should catch that as early as possible